src/SFA.DAS.LevyTransferMatching.Web/Views/Pledges/ViewApplicantTransferPledge.cshtml

```diff
@@ -0,0 +1,90 @@
+﻿@using SFA.DAS.LevyTransferMatching.Web.Extensions
+@model SFA.DAS.LevyTransferMatching.Web.Models.Pledges.GetApplicationViewModel
+@{
+    Model.SetupModel();
+}
+<a href="javascript:history.back()" class="govuk-back-link">Back</a>
```

Comment:
Hi Steve, these back links should link to a specific page rather than the previous page in user history, i.e. the page previous to this one in the user journey. In this case I suppose it should be the Applications list page.

Reply:
noted, cheers

Reply:
changed the text to "Back to my transfer applications"